src/cache/cache_las.c

```diff
@@ -709,25 +723,48 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
         prev_upd = NULL;
```

Comment:
Why don't we retrieve the full value from las here?

Reply:
We don't have the previous updates value to get the stop_time_pair.

Reply:
OK, can we peek the value here? We can add a peek function in modify_vector and get the stop timestamp. I think the code will be cleaner if we can search las here.

Reply:
Agreed will add peek.